cypress/e2e/06-customized-events.cy.ts

```diff
@@ -44,3 +44,22 @@ describe('CustomizedEvents.vue', () => {
     cy.compareSnapshot('06-customized-events-month-flyout')
   })
 })
+
+describe('CustomizedEvents.vue', () => {
+  beforeEach(() => {
+    PageObject.simulateMobileView(); // Ensure mobile view is simulated
+    cy.visit('#/cypress/customized-events');
+  });
+
+  it('Should display a custom agenda event in mobile view', () => {
+    PageObject.setMonthMode(); // This should automatically switch to agenda mode in mobile view
+    PageObject.assertAgendaEventExistsWithId('#agenda__event-custom-id'); // Replace with actual ID
```

Comment:
Yet to be implemented?

Reply:
Yes, my bad. Tomorrow I'll be at my place so I'll have access to my ubuntu machine to run the Cypress tests, will finish this then.

Reply:
Right, that's something I should probably put in the contributing.md file - if you're developing Cypress tests on a machine other than a mac, just comment out all the screenshot statements in the test suites you're running, as long as you comment them back in before pushing.